### PR TITLE
Remove yara from sst_csi_client_tools

### PR DESCRIPTION
**Pull request by mhuth**

### CODE REVIEW

**Comment by mhuth** [comment]
@ptoscano please do the needful.

**Comment by ptoscano** [comment]
While removing it from sst_client_tools is correct, it will drop it entirely from the distro. yara **must** be in a workload to be shipped.

If yara belongs to sst_insights, then please move it to that workload.

**Comment by mhuth** [comment]
@ptoscano It belongs to platmgmt-ins-security which isn't listed as a group in the config directory.  But it is a rover group: https://rover.redhat.com/groups/group/platmgmt-ins-security

**Comment by ptoscano** [comment]
> It belongs to platmgmt-ins-security which isn't listed as a group in the config directory.

As I said, it must belong to a RHEL SST.

File: configs/sst_insights.yaml [hunk 1]
```diff
@@ -4,7 +4,14 @@ data:
...
   maintainer: sst_insights
-  packages: []
+  packages:
+    - yara
+  package_placeholders:
+    - srpm_name: yara
+      rpms:
+        - rpm_name: yara
+          description: yara
+          dependencies: []
```
Comment (yselkowitz) [inline configs/sst_insights.yaml:14]: See https://github.com/minimization/content-resolver-input/pull/1294#pullrequestreview-2353177631

**Review by yselkowitz — CHANGES_REQUESTED** [review]

**Review by yselkowitz — APPROVED** [review]

**Pull request merged**